frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -158,93 +158,24 @@ export default function SurveyEdit(): JSX.Element {
-                                                                                                            'off',
-                                                                                                        glyphMargin:
-                                                                                                            false,
-                                                                                                        folding: false,
-                                                                                                    }}
-                                                                                                />
-                                                                                            </div>
-                                                                                        ),
-                                                                                    },
-                                                                                ]}
-                                                                            />
-                                                                            {question.description &&
-                                                                                question.description
-                                                                                    ?.toLowerCase()
-                                                                                    .includes('<script') && (
-                                                                                    <LemonBanner type="warning">
-                                                                                        Scripts won't run in the survey
-                                                                                        popup and we'll remove these on
-                                                                                        save. Use the API question mode
-                                                                                        to run your own scripts in
-                                                                                        surveys.
-                                                                                    </LemonBanner>
-                                                                                )}
-                                                                        </>
+                                                                        <HTMLEditor
+                                                                            value={value}
+                                                                            onChange={onChange}
+                                                                            showScriptWarning={
+                                                                                !!(
```

Comment:
don't need this parameter, can get it from 'value', and keeps logic in one place